flipperdevices/flipperzero-good-faps
# Picopass: Rename option

# What's new

- Rename support (https://github.com/flipperdevices/flipperzero-good-faps/issues/12)

# Verification 

- Launch app
- Choose "Saved"
- Choose a card
- Choose "Rename"
- Give it a new name
- Confirm new name via file browser

# Checklist (For Reviewer)
